src/extensions/surveys.tsx

```diff
@@ -655,6 +655,7 @@ interface SurveyPopupProps {
     removeSurveyFromFocus: (id: string) => void
     isPopup?: boolean
     onPreviewSubmit?: (res: string | string[] | number | null) => void
+    onSurveyDismissedOrSent?: () => void
```

Comment:
I think we'd benefit more from 2 callbacks.
one for dismissed and one for sent.
sent (show something with a voucher, etc), dismissed (show something else, do nothing).

Reply:
ok, i thought people could use as part of this https://posthog.com/docs/surveys/implementing-custom-surveys